uqcsbot/scripts/hoogle.py

```diff
@@ -11,9 +12,9 @@ def get_endpoint(type_sig: str) -> str:
 
 
 def pretty_hoogle_result(result: dict, is_verbose: bool) -> str:
-    url = result['location']
-    type_sig = result['self']
-    docs = result['docs']
+    url = result['url']
+    type_sig = re.sub('<[^<]+?>', '', result['item'])
+    docs = re.sub('<[^<]+?>', '', result['docs']).replace('\n',' ').replace('&gt;&gt;&gt;','\n>')
```

Comment:
Is this supposed to have a > in the last replace?
`'\n>'`

Reply:
Yeah, this allows the code to be still "quoted" in the slack message, alternatively, it can be removed so that the code isn't quoted.